doc/hardware/sensors.rst

```diff
@@ -0,0 +1,41 @@
+
+Sensors list
+##################################
+
+.. table:: Supported sensors on MinnowBoard MAX, Galileo GEN 2 and Edison boards
+
+    ============== ============================================= ==========================================================================
+    Sensor Name     Sensor Type                                    Comment
+    ============== ============================================= ==========================================================================
+    BMA180          Accel                                          missing driver on Edison
```

Comment:
Does "missing driver..." mean that this sensor is not supported by that platform?  How about changing this to be a column per platform with a "yes" or "no" icon (or text), with blank meaning unknown at this time?   The "yes" entries would then be links (when available) for the underlying document showing how to setup that sensor hardware for that platform (as you've written).